### PR TITLE
Add E306 to docs

### PR DESCRIPTION
**Pull request by spaceone**

### CODE REVIEW

**Comment by spaceone** [comment]
You have ," found 0" also in other messages. I find it good that it is there because this is what actually is printed out.

**Comment by spaceone** [comment]
Flake8 is even better in this as it prints '...' or "name" in italic for variable things.

**Comment by spaceone** [comment]
See http://flake8.pycqa.org/en/latest/user/error-codes.html

File: docs/intro.rst [hunk 1]
```diff
@@ -314,17 +314,17 @@ This is the current list of error and warning codes:
 +------------+----------------------------------------------------------------------+
 | **E3**     | *Blank line*                                                         |
 +------------+----------------------------------------------------------------------+
-| E301       | expected 1 blank line, found 0                                       |
+| E301       | expected 1 blank line, found …                                       |
```
Comment (sigmavirus24) [inline docs/intro.rst]: This now exceeds the stated purpose of this pull request and will break downstream redistributors who don't have an internationalized locale on their builders and CI infrastructure. Please do not use unicode here.

Comment (spaceone) [inline docs/intro.rst]: So, what do you want? just removal of ", found 0"? Where? in E301 or in all E*? Or replacing with "..." (ascii)?

Comment (sigmavirus24) [inline docs/intro.rst]: Let's just stick to adding the new message. Then let's update these in a separate pull request.

**Comment by spaceone** [comment]
https://github.com/PyCQA/pycodestyle/pull/595/commits/349deeae7f03a77954189553f5eace8fe0e980ec

**Comment by sigmavirus24** [comment]
I'll let @IanLee1521 weigh in, but this looks fine to me.

**Comment by IanLee1521** [comment]
Thanks! @spaceone !

**Pull request merged**